modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/network/messages/CmgMessageGroup.java

```diff
@@ -53,4 +53,27 @@ public class CmgMessageGroup {
      * Message type for {@link SuccessResponseMessage}.
      */
     public static final short SUCCESS_RESPONSE = 6;
+
+    /**
+     * Message types for RAFT commands.
+     */
+    public interface Commands  {
```

Comment:
Can you add a description to the commands?

Reply:
Yes